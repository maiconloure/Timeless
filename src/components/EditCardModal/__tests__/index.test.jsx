import React, { useState as useStateMock } from 'react';
import renderer from 'react-test-renderer';

import BacklogCard from '..';

describe('Snapshot', () => {
  it('renders', () => {
    const showEditCard = true;
    const setShowEditCard = jest.fn();

    const tree = renderer
      .create(
        <BacklogCard
          currentCard={{}}
          user={{}}
          getTags={() => {}}
          getDescription={() => {}}
          getDate={() => {}}
          getTime={() => {}}
          currentDate={{}}
          currentTime={{}}
          removeCard={() => {}}
          duplicateCard={() => {}}
          FormataStringData={() => {}}
        />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
