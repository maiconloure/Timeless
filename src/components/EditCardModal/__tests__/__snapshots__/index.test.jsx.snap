// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot renders 1`] = `
<div
  className="sc-hKFyIo kgjUeU"
>
  <div
    className="sc-eCApGN dRfkSR"
  >
    <div
      className="sc-bdnylx fvRbNq"
    >
      <div
        className="sc-gtssRu leLzgb"
      >
        <p>
          DESCRIÇÃO
        </p>
      </div>
      <textarea
        className="sc-dlnjPT gIJSHY"
        onChange={[Function]}
      />
    </div>
    <div
      className="sc-jSFkmK dvDNfj"
    >
      <div
        className="sc-gKAblj icsaFJ"
      >
        <h2
          className="sc-fujyUd dRWgIf"
        >
          ETIQUETAS
        </h2>
        <input
          className="sc-pNWxx cLHdFW"
          onChange={[Function]}
        />
        <div
          className="sc-jrsJCI fFunpv"
        >
          <img
            alt="Ícone de Ampulheta"
            className="sc-kEqYlL jYvYJg"
            onClick={[Function]}
          />
          <span>
            Tempo Execução
             
            <input
              name="time"
              onChange={[Function]}
              style={
                Object {
                  "fontSize": "1rem",
                  "width": "90px",
                }
              }
              type="time"
              value={Object {}}
            />
          </span>
        </div>
        <div
          className="sc-iqAbSa ecUCuf"
        >
          <img
            alt="Ícone de Cronômetro"
            className="sc-kEqYlL jYvYJg"
            onClick={[Function]}
          />
          <span>
            Data Entrega
            <input
              name="date"
              onChange={[Function]}
              style={
                Object {
                  "fontSize": "1rem",
                  "width": "90px",
                }
              }
              type="date"
              value=""
            />
          </span>
        </div>
      </div>
      <div
        className="sc-iCoHVE kLyLlS"
      >
        <h2
          className="sc-fujyUd dRWgIf"
        >
          AÇÕES
        </h2>
        <div
          className="sc-crzoUp fJKmkt"
        >
          <img
            alt="Ícone de Porta"
            className="sc-dIsAE gAEsSU"
            onClick={[Function]}
          />
          <span>
            Mover
          </span>
        </div>
        <div
          className="sc-crzoUp fJKmkt"
          onClick={[Function]}
        >
          <img
            alt="Ícone de Copiar"
            className="sc-dIsAE gAEsSU"
            onClick={[Function]}
          />
          <span>
            Duplicar
          </span>
        </div>
        <div
          className="sc-crzoUp fJKmkt"
          onClick={[Function]}
        >
          <img
            alt="Ícone de Excluir"
            className="sc-dIsAE gAEsSU"
          />
          <span>
            Excluir
          </span>
        </div>
      </div>
    </div>
  </div>
</div>
`;
